DEPLOYING.md

```diff
@@ -0,0 +1,19 @@
+### Timing for Deploys
+
+* Staging: Deployed on successful builds from Master.
+* Production: Deployed after acceptance at standup.
+
+### Pre-Production Checklist
+- [ ] Review the buildpacks on staging and production. Are they the same? If
+  not, make them the same.
+- [ ] Skim the diff between the current production sha and master
+  - `git fetch --all && git diff production/master staging/master`
```

Comment:
maybe provide github URL as well for those who are more comfortable viewing diffs there?

Reply:
like, as a template for them to fill out or as a script to run that actually figures out what the URL should be? (I think both are valid)

Reply:
either works for me!